### PR TITLE
Document Custom Domains on Tencent

### PR DESCRIPTION
**Pull request by MarkvanMents**
Custom Domains can be set on Tencent
This PR documents how this is done.

### CODE REVIEW

**Comment by MarkvanMents** [comment]
@mengdix 
Hi Meng,
This pull request has been around for a long time.
Can you have a quick look and see what else needs to be done to it? I'm not sure if we were waiting for more information to be added, or whether someone else needed to review it.
Thanks
Mark

**Comment by Adam-Dupaski** [comment]
@MarkvanMents @mengdix what's the outlook on verifying and publishing this Tencent content?

File: content/developerportal/deploy/tencent-custom-domains.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+If you already have a signed SSL/TLS certificate, continue with [Uploading Your Own Custom Domain Certificate](#Uploading), below.
+
+## 4 Obtaining a New Signed Certificate
+
+If you do not have an SSL/TLS certificate you can order one from a certificate authority (like GeoTrust, Thawte, Verisign, RapidSSL, GoDaddy, or Comodo). To get a signed SSL/TLS certificate from a certificate authority, you need to provide a *certificate signing request (CSR)*. A private SSL/TLS key and a CSR tied to that key can be created in the Mendix Cloud for you.
+
+### 4.1 Generating a Certificate Request for your Custom Domain{#Generating}
+
+To create a CSR and an RSA (Rivest–Shamir–Adleman) encryption key, follow these steps:
```
Comment (MarkvanMents) [inline content/developerportal/deploy/tencent-custom-domains.md:96]: Can you do this on the Tencent cloud, or do you have to get your Certificate directly and upload it?

File: content/developerportal/deploy/tencent-custom-domains.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+You can also edit an existing custom domain certificate.
+
+![Edit a certificate](attachments/tencent-custom-domains/certificate-edit.png)
+
+{{% alert type="warning" %}}
+For this you will need access to the certificate request that you created for the current certificate.
+{{% /alert %}}
+
+## 5 Configuring a Custom Domain{#Configuring}
+
```
Comment (MarkvanMents) [inline content/developerportal/deploy/tencent-custom-domains.md:197]: Can we remove this section entirely? I think it just covers the same as the Adding Custom Domains section, above?

**Review by MarkvanMents — COMMENTED** [review]
Hi Kyle,
I've made some changes based on your review comments (https://github.com/mendix/docs/pull/3529/commits/073014157c7837d88e593ab7494632f03256f048). Please can you address the additional questions I have, and check if anything else needs to be changed/removed.

File: content/developerportal/deploy/tencent-custom-domains.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+## 5 Frequently Asked Questions
+
+### 5.1 Can I Create a `*.mycompany.com` Wildcard Certificate?
+
+Yes. You can upload your custom domain certificate to all of your apps and use it for all the environments of all of your apps.
+
+You can select the same wildcard certificate per environment by using it with different subdomains. For example, `test.mycompany.com`, `accp.mycompany.com`, and `app.mycompany.com`.
+
+### 5.2 How Do I Construct an Intermediate Certificate Chain Properly?
```
Comment (MarkvanMents) [inline content/developerportal/deploy/tencent-custom-domains.md:133]: Is this relevant - we don't talk about CAs or chains anywhere, just single certificates.

**Comment by MarkvanMents** [comment]
Reviewed by team and can be published.

**Pull request merged**